.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -25,10 +25,10 @@ body:
       required: true
 
   - type: input
```

Comment:
```suggestion
  - type: textarea
```
Could you change this to a `textarea` so that formatting doesn't get messed up when copy-pasting info?

You could also remove the Vercel version input below, now that it is included in the environment info.

Reply:
ah... I thought it was already a textarea! :sweat: 

that's why all current issues have the info on one line :sweat: